payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberViewOnlyController.kt

```diff
@@ -0,0 +1,70 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.VisualTransformation
+import com.stripe.android.model.CardBrand
+import com.stripe.android.ui.core.forms.FormFieldEntry
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.combine
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.map
+
+/**
+ * Controller for the card number field which is view only and never changes.
```

Comment:
I would also add that the CardNumberViewOnlyController will show the field as not enabled, but the value in the initialFieldValue will be returned in the completed FormFieldValue.  If not value is present an empty string will be returned in the completed form field values.

Reply:
Done